package.json

```diff
@@ -78,32 +78,7 @@
     "format": "minimal"
   },
   "resolutions": {
-    "babel-core": "7.0.0-bridge.0",
-    "@babel/code-frame": "^7.0.0",
-    "@babel/core": "^7.0.0",
-    "@babel/generator": "^7.0.0",
-    "@babel/helpers": "^7.0.0",
-    "@babel/highlight": "^7.0.0",
-    "@babel/parser": "^7.0.0",
-    "@babel/plugin-proposal-async-generator-functions": "^7.0.0",
-    "@babel/plugin-proposal-class-properties": "^7.0.0",
-    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
-    "@babel/plugin-proposal-optional-catch-binding": "^7.0.0",
-    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
-    "@babel/plugin-transform-classes": "^7.0.0",
-    "@babel/plugin-transform-destructuring": "^7.0.0",
-    "@babel/plugin-transform-react-constant-elements": "^7.0.0",
-    "@babel/plugin-transform-react-display-name": "^7.0.0",
-    "@babel/plugin-transform-regenerator": "^7.0.0",
-    "@babel/plugin-transform-runtime": "^7.0.0",
-    "@babel/preset-env": "^7.0.0",
-    "@babel/preset-flow": "^7.0.0",
-    "@babel/preset-react": "^7.0.0",
-    "@babel/runtime": "^7.0.0",
-    "@babel/template": "^7.0.0",
-    "@babel/traverse": "^7.0.0",
-    "@babel/types": "^7.0.0",
-    "babel-preset-react-app": "file:./preset-react-app-custom"
+    "babel-eslint": "^9.0.0"
```

Comment:
For some reason one of the `postcss` dependencies is depending on an older `babel-eslint` (???)

Probably a `devDependency` mistakenly published as `dependency`